app/split_integer.py

```diff
@@ -1,5 +1,7 @@
 def split_integer(value: int, number_of_parts: int) -> list:
     parts = []
+    if value <= 1:
+        return [0]
```

Comment:
rollback all changes in `split_integer.py`. You should just write tests for this function